keyboards/southpole/keymaps/foobeard/keymap.c

```diff
@@ -0,0 +1,78 @@
+	*  |  1   |   2  |   3  |  En  | | LShft     |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |    RShft   |   Up   | Mute |
+	*  |------+------+------+------+-+----------------------------------------------------------------------------------------------+--------+------|
+	*  |      0      |   .  |  En  | | Ctrl  |  Win  |  Alt  |                  Space                            |  Fn  |    | Left |  Down  | Right|
+	*  `------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------'
+	*/
+    [_QWERTY] = KEYMAP(
+		KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSLS, KC_DELETE, KC_INS,
+		KC_P7, KC_P8, KC_P9, KC_PPLS, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSPC, KC_VOLU,
+		KC_P4, KC_P5, KC_P6, KC_PENT, KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, KC_VOLD,
+		KC_P1, KC_P2, KC_P3, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_UP, KC_MUTE,
+		KC_P0, KC_PDOT,      KC_LCTL, KC_LGUI, KC_LALT,                              KC_SPC,              RAISE, KC_LEFT, KC_DOWN, KC_RGHT
+	),
+
+	/*layer 1, function layer
+	*  ,--------------------------------------------------------------------------------------------------------------------------------------------.
+	*  |Reset |      |      |      | |  `  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |      |      |Calc  |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |      |      |      |      | |       |      |      |      |      |      |      |      |      |      |      |      |      |           |      |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |      |      |      |      | |        |      |      |      |      |      |      |      |      |      |      |      |                 |      |
+	*  |------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------|
+	*  |      |      |      |      | |           |      |      |      |      |      |      |      |      |      |      |            |        |      |
+	*  |------+------+------+------+-+----------------------------------------------------------------------------------------------+--------+------|
+	*  |             |      |      | |       |       |       |                                                   |      |    |      |        |      |
+	*  `------+------+------+------+-+-------------------------------------------------------------------------------------------------------+------'
+	*/
+	[_RAISE] = KEYMAP(
```

Comment:
Please change this to:

```suggestion
	[_RAISE] = LAYOUT(
```